rust/plugin-lib/src/dispatch.rs

```diff
@@ -97,6 +97,16 @@ impl<'a, P: 'a + Plugin> Dispatcher<'a, P> {
         v.config = conf.unwrap();
     }
 
+    fn do_language_changed(&mut self, view_id: ViewId, new_lang: LanguageId) {
```

Comment:
my thinking is that we would in this case update the view itself, before calling the plugin, sort of like how we do above in `do_did_save`; I'm not sure if with language we will want to pass along what the previous language was, but I think it can't hurt?

we do the config one backwards because it's important to let the plugin know the _set_ of options that are updated, and that seemed like the easiest way to do that. It does feel a bit gross though. 😕

Reply:
I think that in the current version I'm only passing what's appropriate. Please tell me if I'm wrong somewhere.